eng/pipelines/templates/jobs/ci.yml

```diff
@@ -29,7 +29,9 @@ parameters:
 jobs:
   - job: "Build"
     variables:
-      - template: ../variables/globals.yml
```

Comment:
Given you moved the globals.yml to the higher level you should remove the other imports in the other jobs in here as well.

Reply:
I did this for all of them except perf.yml.  Not sure what to do with that one.